src/overlays/actors/ovl_En_Yb/z_en_yb.c

```diff
@@ -466,8 +468,6 @@ void EnYb_Draw(Actor* thisx, PlayState* play) {
             }
             gDPSetEnvColor(POLY_XLU_DISP++, 0, 0, 0, this->alpha);
 
-            if (1) {}
-
```

Comment:
Interesting, I could have sworn I was testing removing all of these after matching but you keep finding them..

Reply:
Like I mentioned in an earlier comment, this was probably required when you first decompiled this. But then something (likely) changed in some call in this function (types, struct types?)